### ISSUE TITLE
Test remote session with Qt4 version

### ISSUE THREAD

**Issue by IvarsKarpics** [mxcube/mxcubeqt#112]
There is a need to test better remote session with Qt4 version.
To be done...

Ivars

**Comment by mguijarr** [comment]
Hello :) Any news on that ? I had a question from Daniele this morning, he wanted to know
if Remote Access is working with MXCuBE 2.2 (Qt 4).

**Comment by IvarsKarpics** [comment]
Hi @mguijarr 

Sorry, I have not tested it on the beamline. I was busy with some beam centering procedures. But I will try to test it next week. More news to come....

**Comment by mguijarr** [comment]
Ok thanks Ivars! Same with me I am very busy... Holidays are slowly coming to save my life, luckily ;)

**Comment by IvarsKarpics** [comment]
Here is a screenshot from my desktop computer:

![mxcube_remote](https://cloud.githubusercontent.com/assets/7237709/17245027/d91bc0ce-5584-11e6-86aa-9e6dba0b4ac0.png)

On the left is master and on the right is a slave. The chat works and access granting also works. I think the problem is when the master mxcube is closed, the slave isntance hangs. This scenario I have to test a bit.
But as we know all is nice and beautiful on the personal computer, but on the beamline is totaly different story. So I will test it next week.

**Comment by IvarsKarpics** [comment]
I did some code clean up and fxed remote connection issues on Qt4 version:
1. If mxcube instance is slave then all widgets appart from InstanceListBrick are disabled.
2. If instance gets control then all widgets are enabled.
3. Fixed widget_synchronize method and added method to the main Bricks. If in the master a widget is changed then it is mirrored in slave. Available widgets that are synchronized are: QLineEdit, QSpinBox, QDoubleSpinBox, QComboBox, QGroupBox 
4. I tested also connection with several slave instances.

More details:
https://github.com/mxcube/BlissFramework/commit/e0ff9405abb7dc5d749468d6fbadbefbb62f6f3e
